cmd/lotus-shed/terminations.go

```diff
@@ -164,7 +164,7 @@ var terminationsCmd = &cli.Command{
 					}
 
 					for _, sector := range sectors {
-						for _, deal := range sector.DealIDs {
+						for _, deal := range sector.DeprecatedDealIDs {
```

Comment:
Also broken, remove it.

Reply:
(or fix it, but that involves digging into the market actor.

Reply:
removed the cmd, if someone wants it back then it's going to look quite different anyway